app/controllers/orders_controller.rb

```diff
@@ -17,11 +19,17 @@ def create
 
       @cart.contents.clear
       flash[:success] = "Order was successfully placed"
-      redirect_to orders_path(order_id: order.id)
+      redirect_to user_orders_path(current_user, order_id: order.id)
     end
   end
 
   def show
     @order = Order.find(params[:id])
   end
+  private
+
+  def require_user
+    render file: 'public/404' unless current_user.id == params[:user_id].to_i
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.